docs/api/protocol-ns.md

```diff
@@ -0,0 +1,309 @@
+# protocolNS
+
+This document describes the new protocol APIs based on NetworkService.
+
+We don't currently have an estimate of when we will enable `NetworkService` by
+default in Electron, but as Chromium is already removing non-`NetworkService`
+code, we might switch before Electron 10.
+
+The content of this document should be moved to `protocol.md` after it is
+determined when to enable `NetworkService` in Electron.
+
+> Register a custom protocol and intercept existing protocol requests.
+
+Process: [Main](../glossary.md#main-process)
+
+An example of implementing a protocol that has the same effect as the
+`file://` protocol:
+
+```javascript
+const { app, protocol } = require('electron')
```

Comment:
Is it actually exposed on `electron.protocol`?  If so the documentation generator might need some modifications because currently it is inferring the include path as `electron.protocolNS`

Reply:
Yup it will be still `electron.protocol` just creates a different class internally based on availability of network service.

Reply:
How about adding a way in documentation generator to ignore the module? For example ignoring titles like `# protocol (Draft)`.

Reply:
@MarshallOfSound I have updated the docs parser to ignore the NetworkService draft docs.